jina/drivers/multimodal.py

```diff
@@ -47,23 +52,25 @@ def positional_modality(self) -> List[str]:
             raise RuntimeError('Could not know which position of the ndarray to load to each modality')
         return self._exec.positional_modality
 
-    def _get_executor_input_arguments(self, content_by_modality: Dict[str, 'np.ndarray']):
-        """
-        From a dictionary ``content_by_modality`` it returns the arguments in the proper order so that they can be
+    def _get_executor_input_arguments(self, content_by_modality: Dict[str, 'np.ndarray']) -> List['np.ndarray']:
+        """From a dictionary ``content_by_modality`` it returns the arguments in the proper order so that they can be
         passed to the executor.
+
          :param content_by_modality: a dictionary of `Document content` by modality name
+         :return: list of input arguments as np arrays
         """
         return [content_by_modality[modality] for modality in self.positional_modality]
 
-    def _apply_all(
+    def apply_all(
```

Comment:
do not change the `name of the function

Reply:
Mistake. Wanted to use PyCharm to create the template, but it only does it for public methods. Reverted

Reply:
Thanks for catching that